docs/source/tutorials/nemo.rst

```diff
@@ -1,8 +1,152 @@
 .. _nemo:
 
 
-NEMO + FABM + ERSEM
-~~~~~~~~~~~~~~~~~~~
+#############################
+NEMO: Atlantic Margin Model
+#############################
+
+This tutorial gives an end-to-end example on how to install and then run
+NEMO-ERSEM on the 7 km Atlantic Margin Model 7 (AMM7) domain using a high performance computing
+(HPC) machine. Here we have used UK National Supercomputing Service
+`ARCHER2 <https://www.archer2.ac.uk/>`__ with a 
+`singularity container <https://sylabs.io/guides/3.5/user-guide/introduction.html>`__ with
+NEMO-ERSEM installed on it.
+
+This tutorial is based on utilising a
+`singularity container <https://sylabs.io/guides/3.5/user-guide/introduction.html>`_ with a NEMO-ERSEM
+install within. The configuration scripts used in the installation can be found
+`here <https://github.com/dalepartridge/AMM7-NEMO4-FABM-setup>`_. You will
+also need access to the shared folder within the ARCHER2 project id `n01` to obtain the forcing
+files.
+
+Obtaining NEMO-ERSEM container
+###########################
+
+The key packages that are installed into 
+`NEMO-ERSEM container <https://github.com/pmlmodelling/NEMO-container>`_ are:
+
+* `NEMO <https://github.com/pmlmodelling/NEMO4.0-FABM>`__
+* `XIOS <http://forge.ipsl.jussieu.fr/ioserver/svn/XIOS/branchs/xios-2.5>`__
+* `FABM <https://github.com/fabm-model/fabm>`__
+* `ERSEM <https://github.com/pmlmodelling/ersem>`__
+
```

Comment:
Suggest a short description in brackets for each package, e.g. NEMO (physical model), XIOS (I/O server), FABM (biogeochemical framework), ERSEM (biogeochemical model) ot something like that.

Reply:
That's a good idea, I will add some short descriptions